language-server/modules/langserver-core/src/test/resources/completion/import_decl/source/import_decl_with_no_module_name1.bal

```diff
@@ -0,0 +1 @@
+import test/
```

Comment:
We can add a new line here and in the other test files

Reply:
Here, not adding a new line is intentional because this will be the state of the source code file when a user is adding an import to an empty bal file.